server/routes/jenkins/tasks.py

```
@@ -0,0 +1,333 @@
+                                conn.commit()
+                                logger.info(
+                                    "[JENKINS][DEPLOY] Triggered RCA for incident %s (task=%s)",
+                                    incident_id, task.id,
+                                )
+                            else:
+                                logger.info("[JENKINS][DEPLOY] RCA rate-limited for user %s", user_id)
+                        except Exception as rca_exc:
+                            logger.exception("[JENKINS][DEPLOY] Failed to trigger RCA: %s", rca_exc)
+
+        except Exception as db_exc:
+            logger.exception("[JENKINS][DEPLOY] DB error: %s", db_exc)
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)